### PR TITLE
[FEATURE] Porting "How to create and edit Expectations with instant feedback from a sample Batch of data" from RTD to Docusaurus.

### PR DESCRIPTION
**Pull request by alexsherstinsky**
…om a sample Batch of data" from RTD to Docusaurus.

Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
-
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: docs/guides/expectations/how-to-create-and-edit-expectations-with-instant-feedback-from-a-sample-batch-of-data.md [hunk 1]
```diff
@@ -2,4 +2,87 @@
...
+
+</Prerequisites>
+
+Steps
+-----
+
+1. The ``--interactive`` mode denotes the fact that you are interacting with your data.  In
+    other words, you have access to a data source and can specify a Batch of data to be used to create Expectations
+    against.  ``--manual`` mode
+    (please see [How to create and edit Expectations based on domain knowledge, without inspecting data directly](./how-to-create-and-edit-expectations-based-on-domain-knowledge-without-inspecting-data-directly))
```
Comment (cdkini) [inline docs/guides/expectations/how-to-create-and-edit-expectations-with-instant-feedback-from-a-sample-batch-of-data.md:19]: Link is broken on Netlify

**Review by cdkini — APPROVED** [review]
One minor link change but LGTM after!

File: docs/guides/expectations/how-to-create-and-edit-expectations-with-instant-feedback-from-a-sample-batch-of-data.md (hunk 1 above)
Comment (alexsherstinsky) [reply docs/guides/expectations/how-to-create-and-edit-expectations-with-instant-feedback-from-a-sample-batch-of-data.md:19]: Great catch -- checking...

Comment (alexsherstinsky) [reply docs/guides/expectations/how-to-create-and-edit-expectations-with-instant-feedback-from-a-sample-batch-of-data.md:19]: @cdkini It works locally!  Any ideas what to do? Reload/retry?..

Comment (alexsherstinsky) [reply docs/guides/expectations/how-to-create-and-edit-expectations-with-instant-feedback-from-a-sample-batch-of-data.md:19]: @cdkini Also seems to work here: `https://deploy-preview-3046--knoxpod.netlify.app/docs/guides/expectations/how-to-create-and-edit-expectations-based-on-domain-knowledge-without-inspecting-data-directly` -- please confirm.

Comment (cdkini) [reply docs/guides/expectations/how-to-create-and-edit-expectations-with-instant-feedback-from-a-sample-batch-of-data.md:19]: Hmm if it works and the build was successful, it's probably a rendering issue? Netlify might just take a second.

Let's give it a few minutes but I think there shouldn't be an issue @alexsherstinsky

Comment (alexsherstinsky) [reply docs/guides/expectations/how-to-create-and-edit-expectations-with-instant-feedback-from-a-sample-batch-of-data.md:19]: 🙇

**Pull request merged**